src/faultlog/deconfig_records.cpp

```diff
@@ -144,11 +144,11 @@ void DeconfigRecords::populate(const GuardRecords& guardRecords,
                 continue;
             }
 
-            ATTR_LOCATION_CODE_Type attrLocCode;
-            if (!DT_GET_PROP(ATTR_LOCATION_CODE, target, attrLocCode))
-            {
-                deconfigJson["LOCATION_CODE"] = attrLocCode;
-            }
+            // getLocationCode checks if attr is present in target else
+            // gets it from partent target
+            ATTR_LOCATION_CODE_Type attrLocCode = {'\0'};
+            openpower::phal::pdbg::getLocationCode(target, attrLocCode);
```

Comment:
I noticed that the pdbg getLocationCode() throws few exceptions. I believe we need to catch them.

Reply:
I guess at LN157 there's a catch block so it should be fine but plz confirm

Reply:
Thank you. I missed it

Reply:
yes that try..catch should be good for rare failures